instrumentation/apache-httpclient/apache-httpclient-5.0/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/ResponsePayloadBridge.java

```diff
@@ -133,18 +133,18 @@ public static boolean isGzipped(Context context) {
 
   private static void checkForGzip(ResponsePayloadBridge bridge) {
     Optional<Header> contentEncodingHeader = Arrays.stream(bridge.headers)
-        .filter(header -> header.getName().equals("Content-Encoding"))
+        .filter(header -> header.getName().equalsIgnoreCase("content-encoding"))
```

Comment:
The other direction <constant>.equals[IgnoreCase](<nullable thing>) is 100% better :-)